pytest_fixtures/api_fixtures.py

```diff
@@ -30,57 +43,53 @@ def module_location(module_org):
     return entities.Location(organization=[module_org]).create()
 
 
-@pytest.fixture(scope='module')
-def module_lce(module_org):
-    return entities.LifecycleEnvironment(organization=module_org).create()
-
-
-@pytest.fixture(scope='module')
-def module_smart_proxy(module_location):
+@pytest.fixture(scope='session')
+def default_smart_proxy():
     smart_proxy = (
         entities.SmartProxy()
         .search(query={'search': 'name={0}'.format(settings.server.hostname)})[0]
         .read()
     )
-    smart_proxy.location.append(entities.Location(id=module_location.id))
-    smart_proxy.update(['location'])
     return entities.SmartProxy(id=smart_proxy.id).read()
 
 
+@pytest.fixture(scope='session')
+def default_domain(default_smart_proxy):
+    _, _, domain_name = settings.server.hostname.partition('.')
```

Comment:
This is actually from the changes in 7804, but I can make it here.

Reply:
I'm going to actually suggest changing this to index in my commit, because all we're binding is `domain_name` to the last item in the tuple. No need to shuffle around the syntax of arguments that we aren't using anyway.

Since we're using partition, and it always returns a three item tuple, I'd actually think explicit unpacking of all three items is better here.

Reply:
the change I suggest combines the unpacking of right value with packing of undesired left values.
i believe jitendra already made this change in his PR though